docs/tutorials/tutorial-jupyter-docker.md

```diff
@@ -193,6 +207,13 @@ as well as the [Python client for Druid](tutorial-jupyter-index.md#python-api-fo
 
 You should now be able to access and complete the tutorials.
 
+## Tutorial setup for ARM-based devices
+1. Start Druid. You can use the [Quickstart (local)](./index.md) instance. The tutorials
```

Comment:
Introduce the procedure. Perhaps:

For ARM-based devices, run Druid outside of the Docker Compose environment as follows: